relay.go

```diff
@@ -217,16 +217,15 @@ func (r *Relayer) handleCallReq(f lazyCallReq) error {
 	}
 	peer := r.peers.GetOrAdd(hostPort)
 
-	ctx, cancel := NewContext(time.Second)
-	defer cancel()
-
-	remoteConn, err := peer.GetConnection(ctx)
+	// TODO: Should connections use the call timeout? Or a separate timeout?
```

Comment:
mmm, I think we should use a separate timeout. Reasonable application-level timeouts are much, much longer than reasonable connection timeouts.